app/models/document.rb

```diff
@@ -48,14 +48,13 @@ def self.find_by_param(content_id_and_locale)
   end
 
   def self.create_initial(content_id: SecureRandom.uuid,
-                          document_type_id:,
+                          document_type_id: nil,
```

Comment:
Why does this now default to `nil`? It's my understanding that a document type is enforced to be set when creating a new document via https://github.com/alphagov/content-publisher/blob/master/app/interactors/new_document/create_interactor.rb#L27

Reply:
So this is because the next PR will be a migration to remove it. What your link does (i think) is to ensure that the user selects a document type. This is still used but is passed to Editions down the line to MetadataRevisions rather than being held in Document. There will be a few small PR's to take this stuff out later today but for the time being this is really just a proof that everything works and we don't use or set `document.document_type` at the moment.

Reply:
Yup I see that, but this still shouldn't default to nil here as it should never be nil. This method is only used from the `NewDocument::CreateInteractor` when creating a new document which will always have a document type, and if not the validation would fail before this code was even ran if that makes sense?

Reply:
So does the `NewDocument::CreateInteractor` not just work with the params? Because if so then we may have to make some changes to it and the edition equivalent and remove it from the document one but as it stands this can be `nil` because as of this pr it's not being set.